### DISCUSSION TITLE
How can I improve the quality of the generated images?

### DISCUSSION THREAD

**Discussion by ClashSAN** [dome272/Paella#10] [Q&A] [answered]
I don't like the outcome of most of the images. Is there something I must do to further refine them?

**Comment by maubreville** [comment, marked as answer]
I guess there is no really easy answer to your question. In general, Paella has not seen as many images as comparative models (like Stable Diffusion) or has been trained with a similar number of steps. So you essentially can't expect images that are similarly good (as we pointed out in the paper).
However, what you could try is to use a model for prompt engineering, such as [this one](https://huggingface.co/succinctly/text2image-prompt-generator?text=Landscape+of+a+small+forest+clearing+in+the+desert).
